xml/System.Numerics/Complex.xml

```diff
@@ -1858,11 +1889,24 @@
         <Parameter Name="right" Type="System.Numerics.Complex" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="left">The double-precision real value to add.</param>
+        <param name="right">The complex value to add.</param>
+        <summary>Adds a double-precision real number to a complex number.</summary>
+        <returns>The sum of <paramref name="left" /> and <paramref name="right" /> as a complex number.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The addition of a real number (which can be regarded as the complex number a + 0i) and a complex number (c + di) takes the following form:
+
+(a + c) + di
+
+Languages that do not support custom operators can call the <xref:System.Numerics.Complex.Add%28System.Double%2CSystem.Numerics.Complex%29?displayProperty=nameWithType> equivalent method instead.
+
+          ]]></format>
+        </remarks>
+        <altmember cref="M:System.Numerics.Complex.Add(System.NDouble,System.Numerics.Complex)" />
```

Comment:
@carlossanlop 
I just realized there is a unintended leading N in `System.NDouble`.

Reply:
Strange, the build passed. I will address it in a new PR. Thanks for noticing.

Reply:
I will correct it in a PR that I am about to send.